Jenkinsfile

```diff
@@ -75,6 +75,9 @@ pipeline {
               }
             }
             stage('Conformance (Truffle)') {
+              options {
+                timeout(time: 60, unit: 'MINUTES')
+              }
```

Comment:
This isn't needed because the top-level stage above it has a timeout of 30 minutes: https://github.com/kframework/evm-semantics/pull/679/files/d768f1d88e03d75ebe36037274fc4e1f7f72b1eb..3d77709323e04c4222fcf39e9fd4a1855b5c123e#diff-58231b16fdee45a03a4ee3cf94a9f2c3R54